lib/window-settings.js

```diff
@@ -0,0 +1,38 @@
+Promise = require('bluebird');
+const electron = require('electron');
+const remote = require('electron').remote;
+const $ = require('jquery');
+const fs = require('fs');
+const dust = Promise.promisifyAll(require('dustjs-linkedin'));
+const path = require('path');
+const libPath = path.join(__dirname, '..', 'lib');
+
+const selectFolders = electron.remote.require(libPath+'/config.js').selectFolders;
+
+const selectedFolders_content = $('#selectedFolders');
+const changeFolders = $('#changeFolders')
+
+changeFolders.click(event => changeFolders_event());
+
+show_selectedFolders();
+
+/**
+ * Invokes the window_settings() function and writes the new path into the glonal var.
+ */
+function changeFolders_event() {
+  var window_settings = remote.getCurrentWindow();
+  global.selectedFolders = selectFolders(window_settings, false);
```

Comment:
Stay away from global variables. The values stored to the config should then be read by the main window.